projects/cycan/tags

```diff
@@ -0,0 +1 @@
+NFT,DeFi,SmartContracts,Exchange,Dex
```

Comment:
please select the one（If you choose too much, the interface will be messy）

Reply:
Solved